lib/diagnostic.js

```diff
@@ -19,16 +19,27 @@ normalizeWords(string);
 
 // 2. Write a function that takes a string argument
 // and returns an array of unique normalized words.
-const uniqueWords = function (/* */) {
-  /* your response here */
+const uniqueWords = function (string) {
+  return let arrayUniqueNormalizedWords = string.split(' ');
 };
 
```

Comment:
On line 23 you you are returning something, defining with `let` and setting it equal to something else. Remember yesterday when we talked about returning lines that *assign* a value? This is why we want to avoid that! ;-)